src/io.c

```diff
@@ -919,15 +919,19 @@ UInt CloseOutputLog ( void )
 **  'OpenOutput' passes  those  file names to 'SyFopen'  like any other name,
 **  they are just a convention between the main and the system package.
 **
-**  It is not neccessary to open the initial output file, 'InitScanner' opens
-**  '*stdout*' for that purpose.  This  file  on the other hand   can not  be
-**  closed by 'CloseOutput'.
+**  The function does nothing and returns success for '*stdout*' and
+**  '*errout*' when 'LockCurrentOutput(1)' is in effect (used for testing
+**  purposes).
```

Comment:
Note: the above paragraph was copied from the corresponding comment in `io.h`, i.e., what I did here was to sync the comment in io.c and io.h

Reply:
Actually, I just checked: even when `InitScanner` existed, it didn't do what was claimed here since about 1997 :-)